### PR TITLE
Feature/update button designs

### PR DESCRIPTION
**Pull request by eskilsen**
Update design of buttons based on figma: https://www.figma.com/file/ytZGjTEFZS5ZIxB5Q7nfQp/2.-Components?node-id=7384%3A4435. Ticket: https://payexjira.atlassian.net/jira/software/c/projects/SWED/boards/312?modal=detail&selectedIssue=SWED-237

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

## Review instructions
[Review instructions](../REVIEW_INSTRUCTIONS.md)

### CODE REVIEW

File: src/App/components/Button/index.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ const Button = ({ label, id, name, className, value, href, icon, loading, type,
 
     return (
         <button className={btnClasses} {...attrs}>{icon ? "\n\t\t" : null}
-            {icon && !iconAfter ? <><i className="material-icons" aria-hidden="true">{icon}</i>{"\n\t"}</> : null}
+            {icon && !iconAfter ? <><i className="material-icons-outlined" aria-hidden="true">{icon}</i>{"\n\t"}</> : null}
```
Comment (PhungNg) [inline src/App/components/Button/index.js:47]: Pixel-jocker incoming!
Margin on left and right side of icon are not equal
image

File: src/less/components/button.less [hunk 2]
```diff
@@ -108,6 +110,7 @@
     &:focus {
         outline: 0;
         box-shadow: 0 0 0 0.1rem @focus-shadow;
+        border-color: @color;
     }
 
     &:hover,
```
Comment (PhungNg) [inline src/less/components/button.less:116]: Probably not this line, but according to Figma, the hover color should be #FBDD91 and not #FFDF88.
The same applies to the focus state.

Comment (PhungNg) [inline src/less/components/button.less:116]: The box shadow for when the Button is active, should now be 1px, with color #FBDD91.

File: src/less/components/button.less [hunk 3]
```diff
@@ -414,25 +422,13 @@
...
+        &.btn-executive-destructive,
         &.btn-guiding-destructive,
         &.btn-link-destructive {
             &:after {
-                border-color: @brand-error-light;
-                border-left-color: @brand-error;
+                border-color: lighten(@brand-secondary, 40%);
+                border-left-color: @brand-secondary;
             }
         }
```
Comment (PhungNg) [inline src/less/components/button.less:434]: Buttons with loading state should not be able to receive an active state
image

Comment (PhungNg) [inline src/less/components/button.less:434]: The After icon size should stay the same regardless of buttons size.
image

Comment (PhungNg) [inline src/less/components/button.less:434]: Executive-destructiv button has the wrong box-shadow color. Should be #EE5058
image

**Review by PhungNg — CHANGES_REQUESTED** [review]
Good work bro!
But seems like you missed some small changes!
Good luck fixing them! Cause by the time you read this, Im long gone friend 😭💔

**Review by m-lilja — APPROVED** [review]
LGTM

**Pull request merged**